aspnetcore/blazor/components/render-modes.md

```diff
@@ -490,3 +512,49 @@ The following example includes the assembly of the `DifferentAssemblyCounter` co
 app.MapRazorComponents<App>()
     .AddAdditionalAssemblies(typeof(DifferentAssemblyCounter).Assembly);
 ```
+
+## Closure of circuits when there are no remaining interactive server components
+
+[!INCLUDE[](~/blazor/includes/closure-of-circuits.md)]
+
+## Custom shorthand render modes
+
+Whatever follows an `@rendermode` directive must be a valid Razor expression that would make sense as a method parameter in C#.
+
+For example, consider how prerendering is disabled. The following expression isn't valid to pass to `@rendermode` because seeking to pass `InteractiveServer(prerender: false)` as a method parameter in `SomeMethod(InteractiveServer(prerender: false))` isn't a valid C# construct for a method parameter of `SomeMethod`.
```

Comment:
This explanation isn't quite right: `InteractiveServer(prerender: false)` isn't valid because `InteractiveServer` isn't a method. It's a static property on the `RenderMode` static class.

Reply:
For ...

> isn't valid because InteractiveServer isn't a method

... and what Steve provided, I'll work on this for the next commit.

For reference, Steve's remarks were ...

> Whatever follows @rendermode has to be a valid expression that would make sense as a method parameter in other code.
>
> For example you can’t do “@rendermode InteractiveServer(prerender: false)” because the following code isn’t valid:
>
> SomeMethod(InteractiveServer(prerender: false));
> 
> What would be valid is:
> 
> SomeMethod (new InteractiveServerRenderMode(prerender: false));
>
> and hence the syntax is “@rendermode new InteractiveServerRenderMode(prerender: false)”
>
> You’ll be able to write “@rendermode InteractiveServer” because we’ll have a “using static” for Microsoft.AspNetCore.Components.Web.RenderMode, and RenderMode.InteractiveServer exists as a static property, and hence:
>
> SomeMethod (InteractiveServer);
>
> … is equivalent to SomeMethod(RenderMode.InteractiveServer), and that’s all valid.